rights-statements.ttl

```diff
@@ -28,202 +30,219 @@
 
 <collection-ic> a skos:Collection ;
```

Comment:
My two cents: if we want to keep it super-clean, then they should be versioned.
But I'd prefer to stay ready to a change about it, as we've not really discussed this beforehand.
I mean, I wouldn't mind if we decide to remove version numbers later if we realize this can be useful for some applications.


On 9/16/15 10:50 PM, Mx A. Matienzo wrote:
> In rights-statements.ttl <https://github.com/rightsstatements/data-model/pull/19#discussion_r39683246>:
>
>> @@ -28,202 +30,219 @@
>>
>>  <collection-ic> a skos:Collection ;
>
>   * Should the collection URIs be versioned? I think they at least need trailing slashes.
>   * Should collections themselves be versioned (with |owl:versionInfo|?)
>
> —
> Reply to this email directly or view it on GitHub <https://github.com/rightsstatements/data-model/pull/19/files#r39683246>.
>